### ISSUE TITLE
After adding setConfig, Location is fetched only one time and then never again, neither on Foreground nor Background, no matter what the config is

### ISSUE THREAD

**Issue by alfredbaudisch** [transistorsoft/react-native-background-geolocation#974]
## Your Environment
* Plugin version: 3.4.2
* Platform: Android
* OS version: 8.0.0 (update: _just updated the system to 9.1.0 and getting the same results_)
* Device manufacturer / model: Huawei P20 Lite, ANE-LX1
* React Native version (`react-native -v`): 0.59.10
* Plugin config
```javascript <!-- Syntax-highlighting:  paste your code below -->
{
  // Geolocation Config
  // @url https://transistorsoft.github.io/react-native-background-geolocation/interfaces/_react_native_background_geolocation_.config.html
  desiredAccuracy: BackgroundGeolocation.DESIRED_ACCURACY_HIGH,

  distanceFilter: 1, // The minimum distance (measured in meters) a device must move horizontally before an update event is generated. However, by default, distanceFilter is elastically auto-calculated by the plugin: When speed increases, distanceFilter increases; when speed decreases, so too does distanceFilter.
  disableElasticity: true,

  stopOnStationary: false,
  stopTimeout: 5, // The number of minutes to wait before turning off location-services after the ActivityRecognition System (ARS) detects the device is STILL
  stopOnTerminate: true, // <-- If false, allow the background-service to continue tracking when user closes the app, but then we need to implement Headless Mode: https://github.com/transistorsoft/react-native-background-geolocation/wiki/Android-Headless-Mode
  startOnBoot: true, // <-- Auto start tracking when device is powered-up.
  disableLocationAuthorizationAlert: true,

  // Debug properties:
  debug: false, // <-- enable this hear sounds for background-geolocation life-cycle.
  logLevel: BackgroundGeolocation.LOG_LEVEL_VERBOSE,

  // Syncing and batching
  // URL and headers are set dynamically in the component with setConfig
  autoSync: true,
  batchSync: true,
  autoSyncThreshold: 10, // The minimum number of persisted records the plugin must accumulate before triggering an autoSync action. Configuring autoSyncThreshold in conjunction with batchSync true can conserve battery by reducing the number of HTTP requests, since HTTP requests consume far more energy / second than GPS.
  maxBatchSize: 100,

  // Request Body template
  httpRootProperty: "logs",
  locationTemplate: '{"lat":<%= latitude %>,"lng":<%= longitude %>,"time":"<%= timestamp %>"}',
}
```

## Expected Behavior
- Keep tracking location on the foreground and background, respecting the supplied config, specially after calling `setConfig` to add dynamic configuration (which changes constantly).
- Upload to my server in batches as per config

## Actual Behavior
- Plugin starts
- I see the Location services notification
- My `onLocation` callback is called and I receive a single location in my server and Location services notification disappears
- Component is updated with user_id and other data then `#setConfig` to reconfigure the plugin with this dynamic data
- Location services notification appears, `onLocation` is called and my server is called again
- Location services notification disappears
- `onLocation` and my HTTP endpoint are never called again, neither in foreground nor background

## Steps to Reproduce

1. This is the component being added to root level of the application:
<details><summary>Component</summary>

```javascript <!-- Syntax-highlighting:  paste your code below -->
const BASE_BG_GEO_CONFIG = {
  // Geolocation Config
  // @url https://transistorsoft.github.io/react-native-background-geolocation/interfaces/_react_native_background_geolocation_.config.html
  desiredAccuracy: BackgroundGeolocation.DESIRED_ACCURACY_HIGH,

  distanceFilter: 1, // The minimum distance (measured in meters) a device must move horizontally before an update event is generated. However, by default, distanceFilter is elastically auto-calculated by the plugin: When speed increases, distanceFilter increases; when speed decreases, so too does distanceFilter.
  disableElasticity: true,

  stopOnStationary: false,
  stopTimeout: 5, // The number of minutes to wait before turning off location-services after the ActivityRecognition System (ARS) detects the device is STILL
  stopOnTerminate: true, // <-- If false, allow the background-service to continue tracking when user closes the app, but then we need to implement Headless Mode: https://github.com/transistorsoft/react-native-background-geolocation/wiki/Android-Headless-Mode
  startOnBoot: true, // <-- Auto start tracking when device is powered-up.
  disableLocationAuthorizationAlert: true,

  // Debug properties:
  debug: false, // <-- enable this hear sounds for background-geolocation life-cycle.
  logLevel: BackgroundGeolocation.LOG_LEVEL_VERBOSE,

  // Syncing and batching
  autoSync: true,
  batchSync: true,
  autoSyncThreshold: 10, // The minimum number of persisted records the plugin must accumulate before triggering an autoSync action. Configuring autoSyncThreshold in conjunction with batchSync true can conserve battery by reducing the number of HTTP requests, since HTTP requests consume far more energy / second than GPS.
  maxBatchSize: 100,

  // Request Body template
  httpRootProperty: "logs",
  locationTemplate: '{"lat":<%= latitude %>,"lng":<%= longitude %>,"time":"<%= timestamp %>"}',
};

export default class ActivityTrackingComponent extends React.Component {
  constructor(props: CompleteProps) {
    super(props);

    this.state = {
      isBgGeoEnabled: false,
      isBgGeoReady: false,
      startedManually: false
    };
  }

  getBgGeoConfigWithDynamicData() {
    const config = {
      ...BASE_BG_GEO_CONFIG,

      // Upload only after the token has been retrieved
      autoSync: this.props.userToken !== null,

      url: this.props.apiHost + batchCreateActivityLogsPath,
      headers: {
        "Authorization": `Bearer ${this.props.userToken ? this.props.userToken : ""}`
      },

      extras: {
        "activity_id": this.props.activeTimerId,
        "user_id": this.props.employeeId
      },
    };

    return config;
  }

  /**
   * Set the BgGeo configuration dynamically as the timers and users changes.
   */
  reconfigureBgGeo() {
    if(this.state.isBgGeoReady) {
      BackgroundGeolocation.setConfig(this.getBgGeoConfigWithDynamicData());
    }
  }

  async configureBackgroundGeolocation() {
    const onLocation = this.onLocation.bind(this);
    const onError = this.onError.bind(this);
    BackgroundGeolocation.onLocation(onLocation, onError);

    // We can't track before calling ready
    BackgroundGeolocation.ready(this.getBgGeoConfigWithDynamicData(), (state) => {
      console.log("- BackgroundGeolocation is configured and ready: ", state.enabled);

      this.setState({
        isBgGeoEnabled: state.enabled,
        isBgGeoReady: true
      });
    });
  }

  componentDidUpdate(prevProps: CompleteProps, prevState: ActivityTrackingComponentState) {
    const isDifferentTimer = prevProps.activeTimerId !== this.props.activeTimerId;
    const isTimerActive = this.props.activeTimerId !== 0;

    if((prevProps.apiHost !== this.props.apiHost) ||
      (prevProps.userToken !== this.props.userToken) ||
      (isDifferentTimer && isTimerActive)) {
      this.reconfigureBgGeo();
    }

    this.setEnabled();
  }

  componentDidMount() {
    this.configureBackgroundGeolocation();
  }

  componentWillUnmount() {
    // this was never called in the tests, the component is kept mounted
    console.log("[ACTIVITY TRACKING UNMOUNTED]"); 
    BackgroundGeolocation.removeListeners();
  }

  onLocation(location : Object) {
    this.props.onLocation(location);
    console.log('[location] -', location);
  }

  onError(error : Object) {
    console.warn('[location] ERROR -', error);
  }

  shouldTrackingBeStarted() {
    return this.props.activeTimerId !== 0;
  }

  startBgGeolocation() {
    this.setState({startedManually: true});
    console.log("[location] START");
    BackgroundGeolocation.start();
  }

  setEnabled() {
    const shouldBeEnabled = this.shouldTrackingBeStarted();

    if(this.state.isBgGeoReady) {
      const actualEnabled = this.state.isBgGeoEnabled;

      if((shouldBeEnabled && !this.state.startedManually) || (actualEnabled !== shouldBeEnabled)) {
        this.setState({isBgGeoEnabled: shouldBeEnabled});

        if(shouldBeEnabled) {
          this.startBgGeolocation();
        } else {
          console.log("[location] STOP");
          BackgroundGeolocation.stop();
        }
      }
    }
  }

  render() {
    if(!this.state.isBgGeoReady) {
      return <ToastError text={"!isBgGeoReady"}/>;
    } else {
      const lastLocation = this.props.coordinates && this.props.lastCoordinatesAt ? this.props.lastCoordinatesAt : "no";
      return <ToastNeutral text={(`user: ${this.props.employeeId}, token: ${!!this.props.userToken}, timer: ${this.props.activeTimerId}, started: ${this.state.startedManually}`) + " - " + (this.state.isBgGeoEnabled ? "enabled" : "disabled") + " - ready: " + (this.state.isBgGeoReady ? "yes" : "no") + " - last location: " + lastLocation}/>;
    }
  }
}
```

</details>

2. The component is not being unmounted, which is expected, so this is not the problem. Also `stop` has never been called from my component manually, which is also expected while I the app is not killed.
3. I tried using aggressive setting as per instructions from your answer in [another issue](https://github.com/transistorsoft/react-native-background-geolocation/issues/680) and I also tried disabling the HTTP sync. The same issue happens
```
{
  distanceFilter: 0,
  locationUpdateInterval: 0,
  fastestLocationUpdateInterval: 0,
  disableElasticity: true,
}
```
4. Huawei device configured according to https://dontkillmyapp.com/huawei
5. I also tried calling `start` again after `setConfig`, but got the same result

## Context

- While users have timers active, track location on both the foreground and background
- Our app has support for multiple logged on users, with persisted sessions, so as soon another user is picked, the token in the plugin's config must be changed with `setConfig`
- Timers/Activities are stopped and started all the time, so I should also be able to keep changing the `activity_id` from the extra params and the tracking should continue.

## Debug logs

<details><summary>Logs</summary>

``` <!-- Syntax highlighting:  DO NOT REMOVE -->
02-05 14:47:28.231 15855 15974 E TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:47:28.231 15855 15974 E TSLocationManager: ║ LICENSE VALIDATION FAILURE: com.___
02-05 14:47:28.231 15855 15974 E TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:47:28.231 15855 15974 E TSLocationManager: ╟─ Failed to find license key in AndroidManifest.  Ensure you've added the key within <application><meta-data android:name="com.transistorsoft.locationmanager.license" android:value="<YOUR LICENSE KEY>" />
02-05 14:47:28.231 15855 15974 E TSLocationManager: ╟─ BackgroundGeolocation is fully functional in DEBUG builds without a license.
02-05 14:47:28.231 15855 15974 E TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:47:33.481 15855 15982 D TSLocationManager: [c.t.l.adapter.TSConfig c] ℹ️   Persist config, dirty: [autoSync, autoSyncThreshold, batchSync, desiredAccuracy, disableElasticity, disableLocationAuthorizationAlert, distanceFilter, extras, fastestLocationUpdateInterval, headers, httpRootProperty, locationTemplate, locationUpdateInterval, logLevel, maxBatchSize, startOnBoot, url]
02-05 14:47:33.515 15855 15982 D TSLocationManager: [c.t.l.a.BackgroundGeolocation ready] LocationPermission :false
02-05 14:48:51.924 15855 15982 D TSLocationManager: [c.t.l.adapter.TSConfig c] ℹ️   Persist config, dirty: [autoSync, extras, headers]
02-05 14:48:51.952 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:48:51.952 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:48:51.952 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 14:48:51.952 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:05.357 15855 15982 D TSLocationManager: [c.t.l.adapter.TSConfig c] ℹ️   Persist config, dirty: [extras, headers]
02-05 14:49:05.385 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:49:05.385 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:05.385 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 14:49:05.385 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:05.394 15855 15982 I TSLocationManager: [c.t.locationmanager.util.b a] 
02-05 14:49:05.394 15855 15982 I TSLocationManager:   🔵  LocationAuthorization: Requesting Background permission
02-05 14:49:07.303 15855 15982 I TSLocationManager: [c.t.locationmanager.util.b a] 
02-05 14:49:07.303 15855 15982 I TSLocationManager:   🔵  LocationAuthorization: Requesting Background permission
02-05 14:49:07.595 15855 15855 I TSLocationManager: [c.t.locationmanager.util.b$a onPermissionGranted] 
02-05 14:49:07.595 15855 15855 I TSLocationManager:   ✅  LocationAuthorization: Permission granted
02-05 14:49:07.600 15855 17186 I TSLocationManager: - Enable: false → true, trackingMode: 1
02-05 14:49:07.601 15855 17185 I TSLocationManager: - Enable: false → true, trackingMode: 1
02-05 14:49:07.607 15855 17185 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:07.607 15855 17185 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:07.607 15855 17186 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:07.607 15855 17186 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:07.635 15855 17186 I TSLocationManager: [c.t.l.g.TSGeofenceManager start] 
02-05 14:49:07.635 15855 17186 I TSLocationManager:   🎾  Start monitoring geofences
02-05 14:49:07.635 15855 17185 I TSLocationManager: [c.t.l.g.TSGeofenceManager start] 
02-05 14:49:07.635 15855 17185 I TSLocationManager:   🎾  Start monitoring geofences
02-05 14:49:07.641 15855 17185 D TSLocationManager: [c.t.l.http.HttpService startMonitoringConnectivityChanges] 
02-05 14:49:07.641 15855 17185 D TSLocationManager:   🎾  Start monitoring connectivity changes
02-05 14:49:07.644 15855 17186 D TSLocationManager: [c.t.l.http.HttpService startMonitoringConnectivityChanges] 
02-05 14:49:07.644 15855 17186 D TSLocationManager:   🎾  Start monitoring connectivity changes
02-05 14:49:07.654 15855 17185 D TSLocationManager: [c.t.locationmanager.device.a c] 
02-05 14:49:07.654 15855 17185 D TSLocationManager:   🎾  Start monitoring powersave changes
02-05 14:49:07.655 15855 17186 D TSLocationManager: [c.t.locationmanager.device.a c] 
02-05 14:49:07.655 15855 17186 D TSLocationManager:   🎾  Start monitoring powersave changes
02-05 14:49:07.671 15855 16060 D TSLocationManager: [c.t.l.http.HttpService a] 
02-05 14:49:07.671 15855 16060 D TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:07.671 15855 16060 D TSLocationManager: ║ 📶  Connectivity change: connected? true
02-05 14:49:07.671 15855 16060 D TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:07.674 15855 17186 I TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringSignificantLocationChanges] 
02-05 14:49:07.674 15855 17186 I TSLocationManager:   🔴  Stop monitoring significant location changes
02-05 14:49:07.684 15855 16060 D TSLocationManager: [c.t.l.http.HttpService a] 
02-05 14:49:07.684 15855 16060 D TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:07.684 15855 16060 D TSLocationManager: ║ 📶  Connectivity change: connected? true
02-05 14:49:07.684 15855 16060 D TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:07.689 15855 17185 D TSLocationManager: [c.t.locationmanager.util.b b] 
02-05 14:49:07.689 15855 17185 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 14:49:07.691 15855 17186 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 14:49:07.691 15855 17186 I TSLocationManager:   🔴  Stop heartbeat
02-05 14:49:07.693 15855 17185 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 14:49:07.693 15855 17185 I TSLocationManager:   🔴  Stop heartbeat
02-05 14:49:07.709 15855 17186 D TSLocationManager: [c.t.locationmanager.util.b b] 
02-05 14:49:07.709 15855 17186 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 14:49:07.715 15855 17186 I TSLocationManager: [c.t.l.service.TrackingService a] 
02-05 14:49:07.715 15855 17186 I TSLocationManager:   🔵  setPace: false → false
02-05 14:49:07.905 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:07.905 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 14:49:08.073 15855 17193 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 14:49:08.073 15855 17193 I TSLocationManager:   ℹ️  Location availability: true
02-05 14:49:08.099 15855 17185 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 14:49:08.099 15855 17185 I TSLocationManager:   ℹ️  Location availability: true
02-05 14:49:08.122 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:08.122 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:08.122 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 14:49:08.122 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:08.122 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 14:49:08.122 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:49:08.376 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 14:49:08.376 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 14:49:08.676 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:49:08.676 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:08.676 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 14:49:08.676 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:08.690 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:49:08.690 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:08.690 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 14:49:08.690 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:09.970 15855 17193 I TSLocationManager: [c.t.l.s.LocationRequestService a] 
02-05 14:49:09.970 15855 17193 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:09.970 15855 17193 I TSLocationManager: ║ motionchange LocationResult: 2
02-05 14:49:09.970 15855 17193 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:09.970 15855 17193 I TSLocationManager: ╟─ 📍  Location[fused 56******,13****** hAcc=17 et=+134d13h38m53s934ms alt=37.29999923706055 vAcc=2 sAcc=??? bAcc=???], age: 118ms, time: 1580910549850
02-05 14:49:09.979 15855 17193 I TSLocationManager: [c.t.l.l.TSLocationManager onSingleLocationResult] 
02-05 14:49:09.979 15855 17193 I TSLocationManager:   🔵  Acquired motionchange position, isMoving: false
02-05 14:49:09.982 15855 17193 D TSLocationManager: [c.t.l.l.TSLocationManager calculateMedianAccuracy] Median accuracy: 17.364
02-05 14:49:09.983 15855 17196 I TSLocationManager: [c.t.l.s.LocationRequestService a] 
02-05 14:49:09.983 15855 17196 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:09.983 15855 17196 I TSLocationManager: ║ getCurrentPosition LocationResult: 1
02-05 14:49:09.983 15855 17196 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:09.983 15855 17196 I TSLocationManager: ╟─ 📍  Location[fused 56******,13****** hAcc=17 et=+134d13h38m53s934ms alt=37.29999923706055 vAcc=2 sAcc=??? bAcc=???], age: 130ms, time: 1580910549850
02-05 14:49:09.990 15855 17196 I TSLocationManager: [c.t.l.l.TSLocationManager onSingleLocationResult] 
02-05 14:49:09.990 15855 17196 I TSLocationManager:   🔵  Acquired current position
02-05 14:49:09.995 15855 17196 D TSLocationManager: [c.t.l.l.TSLocationManager calculateMedianAccuracy] Median accuracy: 17.364
02-05 14:49:10.004 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 14:49:10.041 15855 17196 D TSLocationManager: [c.t.l.s.LocationRequestService a] SingleLocationRequest 1 isFinished? true
02-05 14:49:10.046 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:10.046 15855 15855 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:10.051 15855 15855 D TSLocationManager: [c.t.l.g.TSGeofenceManager startMonitoringStationaryRegion] 
02-05 14:49:10.051 15855 15855 D TSLocationManager:   🎾  Start monitoring stationary region (radius: 150.0m 55.6662915,12.5814847 hAcc=17.364)
02-05 14:49:10.057 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 14:49:10.062 15855 17193 D TSLocationManager: [c.t.l.s.LocationRequestService a] SingleLocationRequest 2 isFinished? true
02-05 14:49:10.072 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:10.072 15855 15855 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:10.074 15855 17185 I TSLocationManager: [c.t.l.data.sqlite.b persist] 
02-05 14:49:10.074 15855 17185 I TSLocationManager:   ✅  INSERT: 3e3b850f-1d4f-4100-afdb-4f40783b20f4
02-05 14:49:10.076 15855 15855 D TSLocationManager: [c.t.l.g.TSGeofenceManager startMonitoringStationaryRegion] 
02-05 14:49:10.076 15855 15855 D TSLocationManager:   🎾  Start monitoring stationary region (radius: 150.0m 55.6662915,12.5814847 hAcc=17.364)
02-05 14:49:10.088 15855 17185 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:49:10.088 15855 17185 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:10.088 15855 17185 I TSLocationManager: ║ HTTP Service (count: 1)
02-05 14:49:10.088 15855 17185 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:10.114 15855 15855 I TSLocationManager: [c.t.l.service.TrackingService h] 
02-05 14:49:10.114 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:10.114 15855 15855 I TSLocationManager: ║ TrackingService motionchange: false
02-05 14:49:10.114 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:10.145 15855 15855 I TSLocationManager: [c.t.l.service.TrackingService h] 
02-05 14:49:10.145 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:10.145 15855 15855 I TSLocationManager: ║ TrackingService motionchange: false
02-05 14:49:10.145 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:10.185 15855 15855 I TSLocationManager: [c.t.l.u.BackgroundTaskManager onStartJob] ⏳ startBackgroundTask: 1
02-05 14:49:10.201 15855 17195 D TSLocationManager: [c.t.l.data.sqlite.b allWithLocking] 
02-05 14:49:10.201 15855 17195 D TSLocationManager:   ✅  Locked 1 records
02-05 14:49:10.203 15855 17195 I TSLocationManager: [c.t.l.http.HttpService a] 
02-05 14:49:10.203 15855 17195 I TSLocationManager:   🔵  HTTP POST batch (1)
02-05 14:49:10.274 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:10.274 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 14:49:10.321 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:10.321 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 14:49:10.363 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:10.363 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:10.363 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 14:49:10.363 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:10.363 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 14:49:10.363 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:49:10.403 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:10.403 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:10.403 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 14:49:10.403 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:10.403 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 14:49:10.403 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:49:10.412 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 14:49:10.412 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 14:49:10.657 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 14:49:10.657 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 14:49:11.189 15855 17223 I TSLocationManager: [c.t.l.http.HttpService$c onResponse] 
02-05 14:49:11.189 15855 17223 I TSLocationManager:   🔵  Response: 200
02-05 14:49:11.198 15855 17223 D TSLocationManager: [c.t.l.data.sqlite.b destroyAll] 
02-05 14:49:11.198 15855 17223 D TSLocationManager:   ✅  DELETED: (1)
02-05 14:49:11.209 15855 17223 D TSLocationManager: [c.t.l.data.sqlite.b allWithLocking] 
02-05 14:49:11.209 15855 17223 D TSLocationManager:   ✅  Locked 0 records
02-05 14:49:11.212 15855 17223 I TSLocationManager: [c.t.l.u.BackgroundTaskManager$Task stop] ⏳ stopBackgroundTask: 1
02-05 14:49:11.217 15855 15855 D TSLocationManager: [c.t.l.s.BackgroundTaskService onDestroy] 
02-05 14:49:24.489 15855 15982 D TSLocationManager: [c.t.l.l.TSLocationManager clearLastOdometerLocation] 
02-05 14:49:24.489 15855 15982 D TSLocationManager:   ℹ️  Clear last odometer location
02-05 14:49:24.509 15855 17192 D TSLocationManager: [c.t.l.g.TSGeofenceManager c] ℹ️  Persist monitored geofences: []
02-05 14:49:24.510 15855 17192 D TSLocationManager: [c.t.l.g.TSGeofenceManager b] 
02-05 14:49:24.510 15855 17192 D TSLocationManager:   🔴  Stop monitoring geofences
02-05 14:49:24.517 15855 15982 I TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringSignificantLocationChanges] 
02-05 14:49:24.517 15855 15982 I TSLocationManager:   🔴  Stop monitoring significant location changes
02-05 14:49:24.527 15855 15982 D TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringStationaryRegion] 
02-05 14:49:24.527 15855 15982 D TSLocationManager:   🔴  Stop monitoring stationary region
02-05 14:49:24.540 15855 15982 I TSLocationManager: [c.t.l.s.ActivityRecognitionService b] 
02-05 14:49:24.540 15855 15982 I TSLocationManager:   🔴  Stop motion-activity updates
02-05 14:49:24.547 15855 15982 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 14:49:24.547 15855 15982 I TSLocationManager:   🔴  Stop heartbeat
02-05 14:49:24.560 15855 15982 D TSLocationManager: [c.t.l.http.HttpService stopMonitoringConnectivityChanges] 
02-05 14:49:24.560 15855 15982 D TSLocationManager:   🔴  Stop monitoring connectivity changes
02-05 14:49:24.576 15855 15982 D TSLocationManager: [c.t.locationmanager.device.a d] 
02-05 14:49:24.576 15855 15982 D TSLocationManager:   🔴  Stop monitoring powersave changes
02-05 14:49:24.586 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 14:49:24.586 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 14:49:24.607 15855 15855 D TSLocationManager: [c.t.l.s.GeofencingService onDestroy] 
02-05 14:49:24.607 15855 15855 D TSLocationManager:   🔴  GeofencingService destroyed
02-05 14:49:24.677 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 14:49:24.677 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 14:49:36.163 15855 15982 D TSLocationManager: [c.t.l.adapter.TSConfig c] ℹ️   Persist config, dirty: [extras, headers]
02-05 14:49:36.185 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:49:36.185 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:36.185 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 14:49:36.185 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:36.188 15855 15982 D TSLocationManager: [c.t.locationmanager.util.b a] 
02-05 14:49:36.188 15855 15982 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 14:49:36.190 15855 17192 I TSLocationManager: - Enable: false → true, trackingMode: 1
02-05 14:49:36.192 15855 17192 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:36.192 15855 17192 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:36.195 15855 17192 I TSLocationManager: [c.t.l.g.TSGeofenceManager start] 
02-05 14:49:36.195 15855 17192 I TSLocationManager:   🎾  Start monitoring geofences
02-05 14:49:36.198 15855 17192 D TSLocationManager: [c.t.l.http.HttpService startMonitoringConnectivityChanges] 
02-05 14:49:36.198 15855 17192 D TSLocationManager:   🎾  Start monitoring connectivity changes
02-05 14:49:36.202 15855 17192 D TSLocationManager: [c.t.locationmanager.device.a c] 
02-05 14:49:36.202 15855 17192 D TSLocationManager:   🎾  Start monitoring powersave changes
02-05 14:49:36.207 15855 16060 D TSLocationManager: [c.t.l.http.HttpService a] 
02-05 14:49:36.207 15855 16060 D TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:36.207 15855 16060 D TSLocationManager: ║ 📶  Connectivity change: connected? true
02-05 14:49:36.207 15855 16060 D TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:36.211 15855 17192 I TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringSignificantLocationChanges] 
02-05 14:49:36.211 15855 17192 I TSLocationManager:   🔴  Stop monitoring significant location changes
02-05 14:49:36.214 15855 17192 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 14:49:36.214 15855 17192 I TSLocationManager:   🔴  Stop heartbeat
02-05 14:49:36.229 15855 17192 D TSLocationManager: [c.t.locationmanager.util.b b] 
02-05 14:49:36.229 15855 17192 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 14:49:36.243 15855 17192 I TSLocationManager: [c.t.l.service.TrackingService a] 
02-05 14:49:36.243 15855 17192 I TSLocationManager:   🔵  setPace: false → false
02-05 14:49:36.282 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:36.282 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 14:49:36.410 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:36.410 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:36.410 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 14:49:36.410 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:36.410 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 14:49:36.410 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:49:36.450 15855 17186 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 14:49:36.450 15855 17186 I TSLocationManager:   ℹ️  Location availability: true
02-05 14:49:36.663 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 14:49:36.663 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 14:49:37.214 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:49:37.214 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:37.214 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 14:49:37.214 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:38.345 15855 17194 I TSLocationManager: [c.t.l.s.LocationRequestService a] 
02-05 14:49:38.345 15855 17194 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:38.345 15855 17194 I TSLocationManager: ║ motionchange LocationResult: 3
02-05 14:49:38.345 15855 17194 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:38.345 15855 17194 I TSLocationManager: ╟─ 📍  Location[fused 56******,13****** hAcc=24 et=+134d13h39m22s280ms alt=37.5 vAcc=2 sAcc=??? bAcc=???], age: 145ms, time: 1580910578196
02-05 14:49:38.352 15855 17194 I TSLocationManager: [c.t.l.l.TSLocationManager onSingleLocationResult] 
02-05 14:49:38.352 15855 17194 I TSLocationManager:   🔵  Acquired motionchange position, isMoving: false
02-05 14:49:38.354 15855 17194 D TSLocationManager: [c.t.l.l.TSLocationManager calculateMedianAccuracy] Median accuracy: 17.364
02-05 14:49:38.384 15855 17194 D TSLocationManager: [c.t.l.s.LocationRequestService a] SingleLocationRequest 3 isFinished? true
02-05 14:49:38.392 15855 17145 I TSLocationManager: [c.t.l.data.sqlite.b persist] 
02-05 14:49:38.392 15855 17145 I TSLocationManager:   ✅  INSERT: eca8a7c2-0e58-4cc9-8195-b2255afdf194
02-05 14:49:38.394 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 14:49:38.397 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:38.397 15855 15855 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:38.401 15855 15855 D TSLocationManager: [c.t.l.g.TSGeofenceManager startMonitoringStationaryRegion] 
02-05 14:49:38.401 15855 15855 D TSLocationManager:   🎾  Start monitoring stationary region (radius: 150.0m 55.666275,12.5814277 hAcc=24.376)
02-05 14:49:38.412 15855 17145 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 14:49:38.412 15855 17145 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:38.412 15855 17145 I TSLocationManager: ║ HTTP Service (count: 1)
02-05 14:49:38.412 15855 17145 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:38.453 15855 15855 I TSLocationManager: [c.t.l.service.TrackingService h] 
02-05 14:49:38.453 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:38.453 15855 15855 I TSLocationManager: ║ TrackingService motionchange: false
02-05 14:49:38.453 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:38.489 15855 15855 I TSLocationManager: [c.t.l.u.BackgroundTaskManager onStartJob] ⏳ startBackgroundTask: 2
02-05 14:49:38.505 15855 17196 D TSLocationManager: [c.t.l.data.sqlite.b allWithLocking] 
02-05 14:49:38.505 15855 17196 D TSLocationManager:   ✅  Locked 1 records
02-05 14:49:38.511 15855 17196 I TSLocationManager: [c.t.l.http.HttpService a] 
02-05 14:49:38.511 15855 17196 I TSLocationManager:   🔵  HTTP POST batch (1)
02-05 14:49:38.665 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:38.665 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 14:49:38.714 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 14:49:38.714 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 14:49:38.757 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:38.757 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:38.757 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 14:49:38.757 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:38.757 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 14:49:38.757 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:49:38.970 15855 17223 I TSLocationManager: [c.t.l.http.HttpService$c onResponse] 
02-05 14:49:38.970 15855 17223 I TSLocationManager:   🔵  Response: 200
02-05 14:49:38.978 15855 17223 D TSLocationManager: [c.t.l.data.sqlite.b destroyAll] 
02-05 14:49:38.978 15855 17223 D TSLocationManager:   ✅  DELETED: (1)
02-05 14:49:38.993 15855 17223 D TSLocationManager: [c.t.l.data.sqlite.b allWithLocking] 
02-05 14:49:38.993 15855 17223 D TSLocationManager:   ✅  Locked 0 records
02-05 14:49:38.995 15855 17223 I TSLocationManager: [c.t.l.u.BackgroundTaskManager$Task stop] ⏳ stopBackgroundTask: 2
02-05 14:49:38.998 15855 15855 D TSLocationManager: [c.t.l.s.BackgroundTaskService onDestroy] 
02-05 14:49:39.012 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 14:49:39.012 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 14:49:40.524 15855 15982 D TSLocationManager: [c.t.locationmanager.util.b a] 
02-05 14:49:40.524 15855 15982 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 14:49:40.528 15855 17186 I TSLocationManager: - Enable: true → true, trackingMode: 1
02-05 14:49:40.530 15855 17186 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:40.530 15855 17186 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:40.535 15855 17186 D TSLocationManager: [c.t.l.http.HttpService startMonitoringConnectivityChanges] 
02-05 14:49:40.535 15855 17186 D TSLocationManager:   🎾  Start monitoring connectivity changes
02-05 14:49:40.535 15855 17186 D TSLocationManager: [c.t.locationmanager.device.a c] 
02-05 14:49:40.535 15855 17186 D TSLocationManager:   🎾  Start monitoring powersave changes
02-05 14:49:40.540 15855 17186 D TSLocationManager: [c.t.locationmanager.util.b b] 
02-05 14:49:40.540 15855 17186 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 14:49:40.541 15855 17186 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 14:49:40.541 15855 17186 I TSLocationManager:   🔴  Stop heartbeat
02-05 14:49:40.686 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:40.686 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 14:49:40.763 15855 17191 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 14:49:40.763 15855 17191 I TSLocationManager:   ℹ️  Location availability: true
02-05 14:49:40.806 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:40.806 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:40.806 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 14:49:40.806 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:40.806 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 14:49:40.806 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:49:41.059 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 14:49:41.059 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 14:49:47.777 15855 17185 I TSLocationManager: [c.t.l.s.LocationRequestService a] 
02-05 14:49:47.777 15855 17185 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:47.777 15855 17185 I TSLocationManager: ║ getCurrentPosition LocationResult: 4
02-05 14:49:47.777 15855 17185 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:47.777 15855 17185 I TSLocationManager: ╟─ 📍  Location[fused 56******,13****** hAcc=17 et=+134d13h39m31s752ms alt=37.5 vel=0.02018666 bear=97.25189 vAcc=2 sAcc=??? bAcc=???], age: 104ms, time: 1580910587668
02-05 14:49:47.781 15855 17185 I TSLocationManager: [c.t.l.l.TSLocationManager onSingleLocationResult] 
02-05 14:49:47.781 15855 17185 I TSLocationManager:   🔵  Acquired current position
02-05 14:49:47.782 15855 17185 D TSLocationManager: [c.t.l.l.TSLocationManager calculateMedianAccuracy] Median accuracy: 17.185001
02-05 14:49:47.791 15855 17185 D TSLocationManager: [c.t.l.s.LocationRequestService a] SingleLocationRequest 4 isFinished? true
02-05 14:49:47.798 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:47.798 15855 15855 I TSLocationManager:   🎾  Start motion-activity updates
02-05 14:49:47.801 15855 15855 D TSLocationManager: [c.t.l.g.TSGeofenceManager startMonitoringStationaryRegion] 
02-05 14:49:47.801 15855 15855 D TSLocationManager:   🎾  Start monitoring stationary region (radius: 150.0m 55.666269,12.5815109 hAcc=17.006)
02-05 14:49:47.846 15855 15855 I TSLocationManager: [c.t.l.service.TrackingService h] 
02-05 14:49:47.846 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:47.846 15855 15855 I TSLocationManager: ║ TrackingService motionchange: false
02-05 14:49:47.846 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:47.979 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:47.979 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 14:49:48.047 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 14:49:48.047 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 14:49:48.047 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 14:49:48.047 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 14:49:48.047 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 14:49:48.047 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 14:49:48.050 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 14:49:48.102 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 14:49:48.102 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 14:49:48.301 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 14:49:48.301 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 15:01:05.414 15855 15855 I TSLocationManager: [c.t.l.s.TSScheduleManager oneShot] 
02-05 15:01:05.414 15855 15855 I TSLocationManager:   ⏰ Scheduled OneShot: TERMINATE_EVENT in 10000ms (jobID: -1708771588)
02-05 15:01:05.667 15855 15982 D TSLocationManager: [c.t.l.a.BackgroundGeolocation c] 
02-05 15:01:05.667 15855 15982 D TSLocationManager:   🔴  Cleared callbacks
02-05 15:01:10.675 15855 22928 D TSLocationManager: [c.t.l.adapter.TSConfig c] ℹ️   Persist config, dirty: [autoSync, distanceFilter, extras, headers]
02-05 15:01:10.689 15855 22928 D TSLocationManager: [c.t.locationmanager.util.b b] 
02-05 15:01:10.689 15855 22928 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 15:01:10.915 15855 22922 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 15:01:10.915 15855 22922 I TSLocationManager:   ℹ️  Location availability: true
02-05 15:01:10.959 15855 22928 D TSLocationManager: [c.t.l.l.TSLocationManager clearLastOdometerLocation] 
02-05 15:01:10.959 15855 22928 D TSLocationManager:   ℹ️  Clear last odometer location
02-05 15:01:10.968 15855 22928 I TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringSignificantLocationChanges] 
02-05 15:01:10.968 15855 22928 I TSLocationManager:   🔴  Stop monitoring significant location changes
02-05 15:01:10.969 15855 22922 D TSLocationManager: [c.t.l.g.TSGeofenceManager c] ℹ️  Persist monitored geofences: []
02-05 15:01:10.974 15855 22928 D TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringStationaryRegion] 
02-05 15:01:10.974 15855 22928 D TSLocationManager:   🔴  Stop monitoring stationary region
02-05 15:01:10.974 15855 22922 D TSLocationManager: [c.t.l.g.TSGeofenceManager b] 
02-05 15:01:10.974 15855 22922 D TSLocationManager:   🔴  Stop monitoring geofences
02-05 15:01:10.974 15855 22928 D TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringStationaryRegion] 
02-05 15:01:10.974 15855 22928 D TSLocationManager:   🔴  Stop monitoring stationary region
02-05 15:01:10.986 15855 22928 I TSLocationManager: [c.t.l.s.ActivityRecognitionService b] 
02-05 15:01:10.986 15855 22928 I TSLocationManager:   🔴  Stop motion-activity updates
02-05 15:01:10.992 15855 22928 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 15:01:10.992 15855 22928 I TSLocationManager:   🔴  Stop heartbeat
02-05 15:01:10.999 15855 22928 D TSLocationManager: [c.t.l.http.HttpService stopMonitoringConnectivityChanges] 
02-05 15:01:10.999 15855 22928 D TSLocationManager:   🔴  Stop monitoring connectivity changes
02-05 15:01:11.001 15855 22928 D TSLocationManager: [c.t.locationmanager.device.a d] 
02-05 15:01:11.001 15855 22928 D TSLocationManager:   🔴  Stop monitoring powersave changes
02-05 15:01:11.075 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 15:01:11.075 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 15:01:11.079 15855 15855 D TSLocationManager: [c.t.l.s.GeofencingService onDestroy] 
02-05 15:01:11.079 15855 15855 D TSLocationManager:   🔴  GeofencingService destroyed
02-05 15:01:11.114 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 15:01:11.114 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 15:01:12.897 15855 23002 I TSLocationManager: [c.t.l.s.LocationRequestService a] 
02-05 15:01:12.897 15855 23002 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:12.897 15855 23002 I TSLocationManager: ║ getCurrentPosition LocationResult: 5
02-05 15:01:12.897 15855 23002 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:12.897 15855 23002 I TSLocationManager: ╟─ 📍  Location[fused 56******,13****** hAcc=18 et=+134d13h50m56s827ms alt=37.5 vAcc=2 sAcc=??? bAcc=???], age: 151ms, time: 1580911272743
02-05 15:01:12.904 15855 23002 I TSLocationManager: [c.t.l.l.TSLocationManager onSingleLocationResult] 
02-05 15:01:12.904 15855 23002 I TSLocationManager:   🔵  Acquired current position
02-05 15:01:12.905 15855 23002 D TSLocationManager: [c.t.l.l.TSLocationManager calculateMedianAccuracy] Median accuracy: 17.006
02-05 15:01:12.924 15855 23002 D TSLocationManager: [c.t.l.s.LocationRequestService a] SingleLocationRequest 5 isFinished? true
02-05 15:01:13.177 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 15:01:15.438 15855 22922 I TSLocationManager: [c.t.l.scheduler.ScheduleEvent onOneShot] 
02-05 15:01:15.438 15855 22922 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:15.438 15855 22922 I TSLocationManager: ║ ⏰ OneShot event fired: TERMINATE_EVENT
02-05 15:01:15.438 15855 22922 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:15.441 15855 22922 D TSLocationManager: [c.t.l.event.TerminateEvent <init>] 
02-05 15:01:15.441 15855 22922 D TSLocationManager:   ℹ️  TERMINATE_EVENT ignored (MainActivity is still active).
02-05 15:01:18.547 15855 22928 D TSLocationManager: [c.t.l.adapter.TSConfig c] ℹ️   Persist config, dirty: [autoSync, extras, headers]
02-05 15:01:18.563 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 15:01:18.563 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:18.563 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 15:01:18.563 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:31.351 15855 22928 D TSLocationManager: [c.t.l.adapter.TSConfig c] ℹ️   Persist config, dirty: [extras, headers]
02-05 15:01:31.364 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 15:01:31.364 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:31.364 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 15:01:31.364 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:31.365 15855 22928 D TSLocationManager: [c.t.locationmanager.util.b a] 
02-05 15:01:31.365 15855 22928 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 15:01:31.367 15855 22922 I TSLocationManager: - Enable: false → true, trackingMode: 1
02-05 15:01:31.370 15855 22922 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:31.370 15855 22922 I TSLocationManager:   🎾  Start motion-activity updates
02-05 15:01:31.374 15855 22922 I TSLocationManager: [c.t.l.g.TSGeofenceManager start] 
02-05 15:01:31.374 15855 22922 I TSLocationManager:   🎾  Start monitoring geofences
02-05 15:01:31.377 15855 22922 D TSLocationManager: [c.t.l.http.HttpService startMonitoringConnectivityChanges] 
02-05 15:01:31.377 15855 22922 D TSLocationManager:   🎾  Start monitoring connectivity changes
02-05 15:01:31.384 15855 22922 D TSLocationManager: [c.t.locationmanager.device.a c] 
02-05 15:01:31.384 15855 22922 D TSLocationManager:   🎾  Start monitoring powersave changes
02-05 15:01:31.421 15855 16060 D TSLocationManager: [c.t.l.http.HttpService a] 
02-05 15:01:31.421 15855 16060 D TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:31.421 15855 16060 D TSLocationManager: ║ 📶  Connectivity change: connected? true
02-05 15:01:31.421 15855 16060 D TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:31.431 15855 22922 I TSLocationManager: [c.t.l.g.TSGeofenceManager stopMonitoringSignificantLocationChanges] 
02-05 15:01:31.431 15855 22922 I TSLocationManager:   🔴  Stop monitoring significant location changes
02-05 15:01:31.434 15855 22922 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 15:01:31.434 15855 22922 I TSLocationManager:   🔴  Stop heartbeat
02-05 15:01:31.443 15855 22922 D TSLocationManager: [c.t.locationmanager.util.b b] 
02-05 15:01:31.443 15855 22922 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 15:01:31.444 15855 22922 I TSLocationManager: [c.t.l.service.TrackingService a] 
02-05 15:01:31.444 15855 22922 I TSLocationManager:   🔵  setPace: false → false
02-05 15:01:31.517 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:31.517 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 15:01:31.642 15855 23003 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 15:01:31.642 15855 23003 I TSLocationManager:   ℹ️  Location availability: true
02-05 15:01:31.657 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:31.657 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:31.657 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 15:01:31.657 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:31.657 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 15:01:31.657 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 15:01:31.913 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 15:01:31.913 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 15:01:32.426 15855 15855 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 15:01:32.426 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:32.426 15855 15855 I TSLocationManager: ║ HTTP Service (count: 0)
02-05 15:01:32.426 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:33.534 15855 23001 I TSLocationManager: [c.t.l.s.LocationRequestService a] 
02-05 15:01:33.534 15855 23001 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:33.534 15855 23001 I TSLocationManager: ║ motionchange LocationResult: 6
02-05 15:01:33.534 15855 23001 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:33.534 15855 23001 I TSLocationManager: ╟─ 📍  Location[fused 56******,13****** hAcc=18 et=+134d13h51m17s511ms alt=37.599998474121094 vAcc=2 sAcc=??? bAcc=???], age: 103ms, time: 1580911293427
02-05 15:01:33.540 15855 23001 I TSLocationManager: [c.t.l.l.TSLocationManager onSingleLocationResult] 
02-05 15:01:33.540 15855 23001 I TSLocationManager:   🔵  Acquired motionchange position, isMoving: false
02-05 15:01:33.542 15855 23001 D TSLocationManager: [c.t.l.l.TSLocationManager calculateMedianAccuracy] Median accuracy: 17.2695
02-05 15:01:33.555 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 15:01:33.562 15855 23001 D TSLocationManager: [c.t.l.s.LocationRequestService a] SingleLocationRequest 6 isFinished? true
02-05 15:01:33.564 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:33.564 15855 15855 I TSLocationManager:   🎾  Start motion-activity updates
02-05 15:01:33.568 15855 15855 D TSLocationManager: [c.t.l.g.TSGeofenceManager startMonitoringStationaryRegion] 
02-05 15:01:33.568 15855 15855 D TSLocationManager:   🎾  Start monitoring stationary region (radius: 150.0m 55.6662686,12.5814512 hAcc=17.767)
02-05 15:01:33.573 15855 23002 I TSLocationManager: [c.t.l.data.sqlite.b persist] 
02-05 15:01:33.573 15855 23002 I TSLocationManager:   ✅  INSERT: 7aeb4db7-7a02-4e93-9370-01efcd8aebdd
02-05 15:01:33.586 15855 23002 I TSLocationManager: [c.t.l.http.HttpService flush] 
02-05 15:01:33.586 15855 23002 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:33.586 15855 23002 I TSLocationManager: ║ HTTP Service (count: 1)
02-05 15:01:33.586 15855 23002 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:33.607 15855 15855 I TSLocationManager: [c.t.l.service.TrackingService h] 
02-05 15:01:33.607 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:33.607 15855 15855 I TSLocationManager: ║ TrackingService motionchange: false
02-05 15:01:33.607 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:33.648 15855 15855 I TSLocationManager: [c.t.l.u.BackgroundTaskManager onStartJob] ⏳ startBackgroundTask: 3
02-05 15:01:33.658 15855 23003 D TSLocationManager: [c.t.l.data.sqlite.b allWithLocking] 
02-05 15:01:33.658 15855 23003 D TSLocationManager:   ✅  Locked 1 records
02-05 15:01:33.659 15855 23003 I TSLocationManager: [c.t.l.http.HttpService a] 
02-05 15:01:33.659 15855 23003 I TSLocationManager:   🔵  HTTP POST batch (1)
02-05 15:01:33.765 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:33.765 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 15:01:33.832 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:33.832 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:33.832 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 15:01:33.832 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:33.832 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 15:01:33.832 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 15:01:33.863 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 15:01:33.863 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 15:01:34.085 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 15:01:34.085 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 15:01:34.962 15855 23213 I TSLocationManager: [c.t.l.http.HttpService$c onResponse] 
02-05 15:01:34.962 15855 23213 I TSLocationManager:   🔵  Response: 200
02-05 15:01:34.969 15855 23213 D TSLocationManager: [c.t.l.data.sqlite.b destroyAll] 
02-05 15:01:34.969 15855 23213 D TSLocationManager:   ✅  DELETED: (1)
02-05 15:01:34.977 15855 23213 D TSLocationManager: [c.t.l.data.sqlite.b allWithLocking] 
02-05 15:01:34.977 15855 23213 D TSLocationManager:   ✅  Locked 0 records
02-05 15:01:34.979 15855 23213 I TSLocationManager: [c.t.l.u.BackgroundTaskManager$Task stop] ⏳ stopBackgroundTask: 3
02-05 15:01:34.982 15855 15855 D TSLocationManager: [c.t.l.s.BackgroundTaskService onDestroy] 
02-05 15:01:35.160 15855 22928 D TSLocationManager: [c.t.locationmanager.util.b a] 
02-05 15:01:35.160 15855 22928 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 15:01:35.163 15855 23002 I TSLocationManager: - Enable: true → true, trackingMode: 1
02-05 15:01:35.164 15855 23002 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:35.164 15855 23002 I TSLocationManager:   🎾  Start motion-activity updates
02-05 15:01:35.168 15855 23002 D TSLocationManager: [c.t.l.http.HttpService startMonitoringConnectivityChanges] 
02-05 15:01:35.168 15855 23002 D TSLocationManager:   🎾  Start monitoring connectivity changes
02-05 15:01:35.169 15855 23002 D TSLocationManager: [c.t.locationmanager.device.a c] 
02-05 15:01:35.169 15855 23002 D TSLocationManager:   🎾  Start monitoring powersave changes
02-05 15:01:35.172 15855 23002 D TSLocationManager: [c.t.locationmanager.util.b b] 
02-05 15:01:35.172 15855 23002 D TSLocationManager:   ℹ️  LocationAuthorization: Permission granted
02-05 15:01:35.177 15855 23002 I TSLocationManager: [c.t.l.service.HeartbeatService a] 
02-05 15:01:35.177 15855 23002 I TSLocationManager:   🔴  Stop heartbeat
02-05 15:01:35.249 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:35.249 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 15:01:35.357 15855 22922 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 15:01:35.357 15855 22922 I TSLocationManager:   ℹ️  Location availability: true
02-05 15:01:35.357 15855 22922 I TSLocationManager: [c.t.l.s.LocationRequestService b] 
02-05 15:01:35.357 15855 22922 I TSLocationManager:   ℹ️  Location availability: true
02-05 15:01:35.374 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:35.374 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:35.374 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 15:01:35.374 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:35.374 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 15:01:35.374 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 15:01:35.627 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 15:01:35.627 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
02-05 15:01:42.389 15855 23216 I TSLocationManager: [c.t.l.s.LocationRequestService a] 
02-05 15:01:42.389 15855 23216 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:42.389 15855 23216 I TSLocationManager: ║ getCurrentPosition LocationResult: 7
02-05 15:01:42.389 15855 23216 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:42.389 15855 23216 I TSLocationManager: ╟─ 📍  Location[fused 56******,13****** hAcc=18 et=+134d13h51m26s331ms alt=38.099998474121094 vel=0.15311523 bear=41.765472 vAcc=2 sAcc=??? bAcc=???], age: 139ms, time: 1580911302247
02-05 15:01:42.394 15855 23216 I TSLocationManager: [c.t.l.l.TSLocationManager onSingleLocationResult] 
02-05 15:01:42.394 15855 23216 I TSLocationManager:   🔵  Acquired current position
02-05 15:01:42.395 15855 23216 D TSLocationManager: [c.t.l.l.TSLocationManager calculateMedianAccuracy] Median accuracy: 17.533
02-05 15:01:42.405 15855 23216 D TSLocationManager: [c.t.l.s.LocationRequestService a] SingleLocationRequest 7 isFinished? true
02-05 15:01:42.417 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:42.417 15855 15855 I TSLocationManager:   🎾  Start motion-activity updates
02-05 15:01:42.423 15855 15855 D TSLocationManager: [c.t.l.g.TSGeofenceManager startMonitoringStationaryRegion] 
02-05 15:01:42.423 15855 15855 D TSLocationManager:   🎾  Start monitoring stationary region (radius: 150.0m 55.666278,12.581466 hAcc=17.656)
02-05 15:01:42.471 15855 15855 I TSLocationManager: [c.t.l.service.TrackingService h] 
02-05 15:01:42.471 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:42.471 15855 15855 I TSLocationManager: ║ TrackingService motionchange: false
02-05 15:01:42.471 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:42.555 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:42.555 15855 15855 D TSLocationManager:   🚘 ️DetectedActivity [type=STILL, confidence=100]
02-05 15:01:42.637 15855 15855 I TSLocationManager: [c.t.l.s.ActivityRecognitionService a] 
02-05 15:01:42.637 15855 15855 I TSLocationManager: ╔═════════════════════════════════════════════
02-05 15:01:42.637 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 15:01:42.637 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:42.637 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 15:01:42.637 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
02-05 15:01:42.658 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 15:01:42.729 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 15:01:42.729 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 15:01:42.892 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 15:01:42.892 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed

```

</details>

**Comment by christocracy** [comment]
Until the plugin shows a `Motion Transition Result` of **not** `still`, it's not going to initiate tracking (eg: `on_foot`, `in_vehicle`.

```
╔═════════════════════════════════════════════
02-05 15:01:42.637 15855 15855 I TSLocationManager: ║ Motion Transition Result
02-05 15:01:42.637 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
02-05 15:01:42.637 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
02-05 15:01:42.637 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
```

The device needs to *Physically move* before tracking engages.

You can simulate `on_foot` by *vigorously shaking the device*.  Can take several minutes on some devices; other devices take just a few seconds.

**Comment by christocracy** [comment]
And Huawei are the worst possible device to test upon.  I own a *Huawei P20 Lite* among my pile of 11 test devices.  It's the worst.

**Comment by alfredbaudisch** [comment]
> Until the plugin shows a `Motion Transition Result` of **not** `still`, it's not going to initiate tracking (eg: `on_foot`, `in_vehicle`.
> 
> ```
> ╔═════════════════════════════════════════════
> 02-05 15:01:42.637 15855 15855 I TSLocationManager: ║ Motion Transition Result
> 02-05 15:01:42.637 15855 15855 I TSLocationManager: ╠═════════════════════════════════════════════
> 02-05 15:01:42.637 15855 15855 I TSLocationManager: ╟─ 🎾  ENTER: still
> 02-05 15:01:42.637 15855 15855 I TSLocationManager: ╚═════════════════════════════════════════════
> ```
> 
> The device needs to _Physically move_ before tracking engages.
> 
> You can simulate `on_foot` by _vigorously shaking the device_. Can take several minutes on some devices; other devices take just a few seconds.

I walked around the block after that part, I just didn't get the log of that because I was disconnected from my laptop. I would boot the application and then walk around.

After `╟─ 🎾  ENTER: still` there is this which says `c.t.l.s.LocationRequestService onDestroy` ànd ` ActivityRecognitionService destroyed`. Is it normal and then it gets created again when it is necessary or is this the source of the issue?

```
02-05 15:01:42.658 15855 15855 D TSLocationManager: [c.t.l.s.LocationRequestService onDestroy] 
02-05 15:01:42.729 15855 15855 D TSLocationManager: [c.t.l.service.TrackingService onDestroy] 
02-05 15:01:42.729 15855 15855 D TSLocationManager:   🔴  TrackingService destroyed
02-05 15:01:42.892 15855 15855 D TSLocationManager: [c.t.l.s.ActivityRecognitionService onDestroy] 
02-05 15:01:42.892 15855 15855 D TSLocationManager:   🔴  ActivityRecognitionService destroyed
```

> And Huawei are the worst possible device to test upon. I own a Huawei P20 Lite among my pile of 11 test devices. It's the worst.

I will see if I can try with another device.

**Comment by christocracy** [comment]
Yes, those service destroyed messages are normal and expected.

My Huawei can take 1000 meters or more to trigger.  Most other devices trigger with just a few meters.

For Huawei, a walk around the block won’t cut it.

**Comment by christocracy** [comment]
Btw, the plugin stores logs in its database.  Than can easily be retrieved with #emailLog function.  See api docs.

**Comment by alfredbaudisch** [comment]
I tried now using the Huawei P20 Lite with:
```
  distanceFilter: 1,
  disableElasticity: true,
```
And it took around 500m of walking before it activated, just like you said. But after it activated, it started tracking like crazy, respecting the distance filter.

That's it, it's working as it should then. Thanks for all the answers.
